components/finances-section/societe-bilans/index.tsx

```diff
@@ -0,0 +1,107 @@
+'use client';
+
+import FAQLink from '#components-ui/faq-link';
+import { AsyncDataSectionClient } from '#components/section/data-section/client';
+import { FullTable } from '#components/table/full';
+import { EAdministration } from '#models/administrations/EAdministration';
+import { IUniteLegale } from '#models/core/types';
+import { ISession } from '#models/user/session';
+import { formatCurrency, formatDate, getDateFromYYYYMM } from '#utils/helpers';
+import { APIRoutesPaths } from 'app/api/data-fetching/routes-paths';
+import { useAPIRouteData } from 'hooks/fetch/use-API-route-data';
+
+export const FinancesSocieteBilansSection: React.FC<{
+  uniteLegale: IUniteLegale;
+  session: ISession | null;
+}> = ({ uniteLegale, session }) => {
+  const banqueDeFranceBilansProtected = useAPIRouteData(
+    APIRoutesPaths.EspaceAgentBilansProtected,
+    uniteLegale.siren,
+    session
+  );
+
+  return (
+    <AsyncDataSectionClient
+      title="Bilans Banque de France"
+      id="bilans-banque-de-france"
+      sources={[EAdministration.MEF]}
```

Comment:
Source is not right. We should add a new administration banque de france

Reply:
Done !